lib/cdo/script_constants.rb

```diff
@@ -134,6 +134,12 @@ module ScriptConstants
       CSD5_NAME = 'csd5-2017'.freeze,
       CSD6_NAME = 'csd6-2017'.freeze,
     ],
+    # Currently only used for TTS.
+    csp_2019: [
```

Comment:
is this actually being used?

Reply:
Nope! I just got confused because the CSD 2019 script names are in here, so I thought I needed these too. Looks like CSD is for some maker changes I didn't know about. Thanks for catching.

Reply:
If I don't include it, a bunch of tests break, so yes?